python/strings_udf/strings_udf/_lib/cpp/strings_udf.pxd

```diff
@@ -11,9 +11,19 @@ from cudf._lib.cpp.types cimport size_type
 from rmm._lib.device_buffer cimport DeviceBuffer, device_buffer
 
 
+cdef extern from "cudf/strings/udf/udf_string.hpp" namespace \
+        "cudf::strings::udf" nogil:
+    cdef cppclass udf_string
+
 cdef extern from "cudf/strings/udf/udf_apis.hpp"  namespace \
         "cudf::strings::udf" nogil:
     cdef unique_ptr[device_buffer] to_string_view_array(column_view) except +
+    cdef unique_ptr[column] column_from_udf_string_array(
+        void*, size_t
```

Comment:
```suggestion
        udf_string* d_strings, size_type size
```
These parameters were changed and match the ones in `free_udf_string_array` now.